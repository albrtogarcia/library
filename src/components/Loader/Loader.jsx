import './Loader.scss'

export const Loader = () => <h1>Loading</h1>
