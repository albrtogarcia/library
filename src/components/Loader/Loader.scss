.alert {
  background-color: transparent;
  color: inherit;
  border: 1px solid transparent;
  padding: 1rem;
  border-radius: .25rem;
  display: flex;
  gap: .5rem;

  .icon {
    width: 1.5rem;
    height: 1.5rem;
  }

  &-title {
    font-size: 1.15rem;
    font-weight: 600;
    line-height: 1.5rem;
    margin: 0;
  }

  &-message {
    font-size: 1rem;
    font-weight: 400;
    line-height: 1.5rem;
    margin: 0;
  }

  &.alert-danger {
    background-color: var(--brand-red, #E31C3D);
    border-color: var(--brand-red, #E31C3D);
    color: var(--white, #FFF);
  }

  &.alert-warning {
    background-color: var(--brand-yellow, #FFC20E);
    border-color: var(--brand-yellow, #FFC20E);
    color: var(--white, #FFF);
  }

  &.alert-info {
    background-color: var(--brand-blue, #0189D5);
    border-color: var(--brand-blue, #0189D5);
    color: var(--white, #FFF);
  }

  &.alert-success {
    background-color: var(--brand-green, #00A859);
    border-color: var(--brand-green, #00A859);
    color: var(--white, #FFF);
  }
}