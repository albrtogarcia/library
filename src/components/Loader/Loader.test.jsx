import { describe, expect, it } from 'vitest'
import { Loader } from './Loader.jsx'

describe('Loader', () => {
  it('matches snapshot', () => {
    const element = Loader()
    expect(element).toMatchSnapshot()
  })
})
