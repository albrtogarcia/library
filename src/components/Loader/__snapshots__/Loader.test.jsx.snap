// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Loader > matches snapshot 1`] = `
<Blueprint5.Spinner
  intent="primary"
/>
`;
